.changeset/react-preserve-data-aria-attrs.md

```
@@ -0,0 +1,14 @@
+---
+"@thesvg/react": patch
+---
+
+Preserve data-* and aria-* attributes in generated React components
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)